worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -291,9 +291,12 @@ private void deleteExistingDirectory(Path outputDirPath, Job job) {
     private FilenameFilter getFilenameFilter() {
         return (dir, name) -> {
             final String filename = name.toLowerCase();
-            final String ndjson = NDJSON.getSuffix();
-            final String zip = ZIP.getSuffix();
-            return filename.endsWith(ndjson) || filename.endsWith(zip);
+            for (FileOutputType type : FileOutputType.values()) {
+                if (filename.endsWith(type.getSuffix())) {
```

Comment:
Confirming that UNKNOWN will only match and empty file name.   In other words, UNKNOWN never matches.  Correct?

Reply:
Good catch